dvc/cache/base.py

```diff
@@ -89,7 +89,29 @@ def load_dir_cache(self, hash_info):
 
         return DirInfo.from_list(d)
 
-    def changed(self, path_info, hash_info):
+    def _get_filtered_hash_info(
+        self, filtered_path_info, hash_info, hash_origin
```

Comment:
Doesn't seem like we need to introduce alternative names, only makes it more confusing
```suggestion
        self, filter_info, hash_info, path_info
```